packages/esm-patient-medications-app/src/add-drug-order/drug-order-form.component.tsx

```diff
@@ -227,9 +264,16 @@ export function DrugOrderForm({ initialOrderBasketItem, onSave, onCancel, prompt
       indication: initialOrderBasketItem?.indication,
       frequency: initialOrderBasketItem?.frequency,
       startDate: defaultStartDate,
+      requireOutpatientQuantity: true,
     },
   });
 
+  useEffect(() => {
+    if (config.checkRequireOutpatientQuantityProperty) {
```

Comment:
As @brandones  suggests, we don't need the front end config here, just use the global property.  For existing consistently, we can default to a value of "true" (ie require outpatient quantity properties) if the GP is not defined.